.env.example

```diff
@@ -26,3 +26,4 @@ AP_REDIS_PORT=6379
 AP_FLOW_TIMEOUT_SECONDS=600
 AP_TELEMETRY_ENABLED=true
 AP_TEMPLATES_SOURCE_URL="https://cloud.activepieces.com/api/v1/flow-templates"
+AP_ISSUE_ARCHIVE_DAYS=1
```

Comment:
Remove this, there is default value already provided in system.ts, use that place to provide a default value and I would suggest something like 7 days as start.